src/layouts/Main.astro

```diff
@@ -223,3 +251,19 @@ const githubEditUrl = `https://github.com/snowpackjs/astro-docs/blob/main/${curr
     </main>
   </body>
 </html>
+
+<script>
+    const btn = document.querySelector('#menu-toggle')
+    const sidebar = document.querySelector('#sidebar-nav')
```

Comment:
Alternate option is to use `getElementById('menu-toggle');`

It's not a massive difference, but general one should opt for the most "clear" solution as this article states:
https://careerkarma.com/blog/javascript-queryselector-vs-getelementbyid/

Reply:
Noted, Ill have that changed man